eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/ZmqWatcher.scala

```diff
@@ -161,11 +164,13 @@ object ZmqWatcher {
 
   def apply(chainHash: ByteVector32, blockCount: AtomicLong, client: ExtendedBitcoinClient): Behavior[Command] =
     Behaviors.setup { context =>
-      context.system.eventStream ! EventStream.Subscribe(context.messageAdapter[NewBlock](b => ProcessNewBlock(b.block)))
+      context.system.eventStream ! EventStream.Subscribe(context.messageAdapter[NewBlock](b => ProcessNewBlock(b.blockHash)))
       context.system.eventStream ! EventStream.Subscribe(context.messageAdapter[NewTransaction](t => ProcessNewTransaction(t.tx)))
       Behaviors.withTimers { timers =>
         // we initialize block count
         timers.startSingleTimer(TickNewBlock, TickNewBlock, 1 second)
+        // we start a timer in case we don't receive ZMQ block events
+        timers.startSingleTimer(TickBlockTimeout, TickBlockTimeout, blockTimeout)
```

Comment:
Nit (does the same):

```suggestion
        timers.startSingleTimer(TickNewBlock, 1 second)
        // we start a timer in case we don't receive ZMQ block events
        timers.startSingleTimer(TickBlockTimeout, blockTimeout)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1910/commits/46935832f41f5a6b4a1c8ce4d57e7acf0844815d